CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ endif()
 
 # Check for toolchain mismatch, user might need to rebuild toolchain
 set(GCC_VERSION "12.2.0")
-set(LLVM_VERSION "15.0.0")
+set(LLVM_VERSION "15.0.3")
```

Comment:
Do we maybe want to turn this into a `15.*` check (at least for this cycle), considering that not everyone cares about ports as much as I do?

Reply:
I think one of the things this used to be used for was an actual path to some libs or headers, though I think we've removed all the fun -nodefaultlibs from the repo

Reply:
It doesn't seem like it is currently used in any logic apart from checking that the toolchain version matches what is expected.